Modules/Filtering/ImageFilterBase/include/itkCastImageFilter.h

```diff
@@ -125,7 +125,7 @@ class ITK_TEMPLATE_EXPORT CastImageFilter:
 
 protected:
   CastImageFilter();
-  virtual ~CastImageFilter() = default;
+  virtual ~CastImageFilter() override = default;
```

Comment:
virtual and override should not both be used.

if override is used, then virtual should be removed.